<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFXEditorClass</class>
 <widget class="QMainWindow" name="FFXEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">FFXEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
border-image: url(:/FFXEditor/Resources/Final_Fantasy_X_Logo.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="menuBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>208</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>208</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:1px;
border-style:solid;</string>
    </property>
   </widget>
   <widget class="QLabel" name="aboutLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>5</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QFrame" name="containerMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>28</y>
      <width>640</width>
      <height>457</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>457</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>457</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#containerMain
{
margin:0 0 0 0;
}</string>
    </property>
   </widget>
   <widget class="QToolButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>5</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Save</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="reloadButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>5</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string notr="true">Reload</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>486</y>
      <width>642</width>
      <height>76</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 0 0 0 10;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="documentTitle">
     <string notr="true"/>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string notr="true"/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ffxeditor.qrc"/>
 </resources>
 <connections/>
</ui>
